persistence/src/commonMain/kotlin/com/wire/kalium/persistence/server_config/ServerConfigStorage.kt

```diff
@@ -0,0 +1,81 @@
+package com.wire.kalium.persistence.server_config
+
+import com.wire.kalium.persistence.kmm_settings.KaliumPreferences
+import com.wire.kalium.persistence.model.ServerConfigEntity
+import kotlinx.serialization.Serializable
+import kotlin.jvm.JvmInline
+
+interface ServerConfigStorage {
+    fun storeConfig(serverConfig: ServerConfigEntity): ServerConfigEntity
```

Comment:
tiny naming suggestion: What do you think of dropping this `config` postfix in the names of the methods?